project_proposal.Rmd

```diff
@@ -0,0 +1,23 @@
+---
+title: "project_proposal"
+author: "Neel Phaterpekar"
+date: "20/11/2020"
+output: html_document
+---
+
+
+
+
+## 3. Analysis Plan
+
+We will be performing a two-tailed hypothesis test comparing the difference in the ratio of the daily increase in cases to the daily tests performed. We have coined this term as the 'response ratio'. Prior to hypothesis testing, we will perform extensive exploratory data analysis to determine what the best measure of central tendency is within our two groups (Canada and USA). We will create a new response ratio column in the data set by dividing the daily new cases by the number of test on that date. We will explore the distribution of the data to check our samples are normally distributed and to ensure that we have sufficient observations to obtain a strong sample size. Our null hypothesis will be that there is no difference in the mean response ratio between Canada and USA (ie the ratios are equal). Our alternative hypothesis will be that the ratios are not equal. Furthermore, we will construct a 95% confidence interval to indicate the range that our sample estimate falls under. 
```

Comment:
1. Do we know a-priori if it'll be a daily `increase`? Would `daily new cases` be a better choice instead?
2. I like the Null and Alternate hypotheses, should we also include mathematical equations for the same?
3. Should we specify the type of test and the test statistic?

Reply:
1. Agreed. Will fix.
2. Not sure if this is necessary for the proposal, presumably we will be talking about this in our actual report. 
3. I specified two-tailed hypothesis, is there something else you had in mind? I will be more specific about the test statistic.